configs/cmt/cmt_small_ascend.yaml

```diff
@@ -0,0 +1,61 @@
+# system
+mode: 0
+distribute: True
+num_parallel_workers: 8
+val_while_train: True
+
+# dataset
+dataset: "imagenet"
+data_dir: "/path/to/imagenet"
+shuffle: True
+dataset_download: False
+batch_size: 128
+drop_remainder: True
+
+# augmentation
+image_resize: 224
+scale: [0.08, 1.0]
+ratio: [0.75, 1.333]
+re_value: "random"
+hflip: 0.5
+interpolation: "bicubic"
+auto_augment: "randaug-m9-mstd0.5"
+aug_repeats: 3
+re_prob: 0.25
+crop_pct: 0.9
+mixup: 0.8
+cutmix: 1.0
+
+# model
+model: "cmt_small"
+num_classes: 1000
+pretrained: False
+ckpt_path: ""
+keep_checkpoint_max: 10
+ckpt_save_dir: "./ckpt"
+epoch_size: 300
+drop_path_rate: 0.1
+dataset_sink_mode: True
+amp_level: "O2"
+
+# loss
+loss: "ce"
+label_smoothing: 0.1
+
+# lr scheduler
+scheduler: "cosine_decay"
+lr: 0.002
+min_lr: 0.00001
+lr_epoch_stair: True
+decay_epochs: 295
+warmup_epochs: 5
+
+# optimizer
+opt: "adamw"
+filter_bias_and_bn: True
+momentum: 0.9
+weight_decay: 0.05
+loss_scale_type: 'dynamic'
+loss_scale: 16777216.0
```

Comment:
This is really large. How can you get this magic number?

Reply:
I use the initial loss scale(2**24) in mindspore.amp.DynamicLossScaleManager. 
![dynamiclossscale](https://github.com/mindspore-lab/mindcv/assets/77485245/ba0d51f9-672a-4bcc-b459-314f4f0d8bc4)